src/components/sbb-divider/sbb-divider.scss

```diff
@@ -0,0 +1,25 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --_divider-color: var(--sbb-color-iron-default);
+  --_divider-background-color: var(--footer-divider-color, var(--sbb-color-cloud-default));
+}
+
+.sbb-divider {
+  display: block;
+
+  &--horizontal {
+    border-top: #{px-to-rem-build(1)} solid var(--_divider-color);
+  }
+
+  &--vertical {
+    height: 100%;
+    border-left: 0.5px solid var(--_divider-color);
+  }
+
+  &--primary-negative {
+    background: var(--sbb-color-black-default);
```

Comment:
- Background not necessary.
- To toggle negative mode, use css var toggling after :host{} entry (divider-color). See https://github.com/lyne-design-system/lyne-components/blob/master/CODING_STANDARDS.md#use-css-variables

Reply:
Unused background color removed 👍